### PR TITLE
Add an API for migration tool

### PR DESCRIPTION
**Pull request by shomatan**
Migration tool requires a delete attachment API for import.
Multiple parameter support for DELETE.

### CODE REVIEW

File: src/main/java/com/nulabinc/backlog4j/api/ImportMethods.java [hunk 1]
```diff
@@ -18,6 +20,8 @@
 
     Issue importUpdateIssue(ImportUpdateIssueParams params) throws BacklogException;
 
+    Attachment importDeleteAttachment(Object issueIdOrKey, Object attachmentId, ImportDeleteAttachmentParams params) throws BacklogException;
```
[the remark this answers is not in the document]
Comment (shomatan) [reply src/main/java/com/nulabinc/backlog4j/api/ImportMethods.java]: I think `importDeleteAttachmentFile`  is better than `importDeleteAttachment`.
Is it OK? @kimzo

[the remark this answers is not in the document]
Comment (kimzo) [reply src/main/java/com/nulabinc/backlog4j/api/ImportMethods.java]: Ah, I think it is not good to contain multiple verbs (import and delete).
import するメソッドであれば、その対象は名詞になっているほうがいいなと。
importなのか削除するメソッドなのか一瞬迷うので。 @shomatan

**Comment by kimzo** [comment]
test comment for webhook

**Comment by kimzo** [comment]
test comment for webhook 2

**Pull request merged**